### PR TITLE
Use shutil.unpack_archive to extract app source

### PR DESCRIPTION
**Pull request by tkdchen**
Signed-off-by: Chenxiong Qi <cqi@redhat.com>

This proposes an alternative way to simplify the code to extract app source to request bundle directory.

### CODE REVIEW

File: cachito/workers/utils.py [hunk 1]
```diff
@@ -1,17 +1 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (athos-ribeiro) [inline cachito/workers/utils.py:1]: Any reasons for not removing the entire module?

Comment (tkdchen) [reply cachito/workers/utils.py:1]: Just thought `utils.py` might be also used to include other utility code in the future.

Comment (lcarva) [reply cachito/workers/utils.py:1]: It's easy enough to add it back in. I'd prefer if we removed it.

Comment (tkdchen) [reply cachito/workers/utils.py]: Done. The empty files are removed.

**Pull request merged**